docs/agent/config/v2.mdx

```diff
@@ -478,50 +478,50 @@ You can configure a single ngrok agent to tunnel to multiple services within a s
+| [`policy.inbound.actions.type`](/docs/http/traffic-policy/actions/index.mdx)                     | String                                                                       | `policy` has been renamed to `traffic_policy` in agent version 3.14.0.                                                                                                                                                                                                                            |
+| [`policy.inbound.actions.config`](/docs/http/traffic-policy/actions/index.mdx)                   | custom                                                                       | `policy` has been renamed to `traffic_policy` in agent version 3.14.0.                                                                                                                                                                                                                            |
+| [`policy.outbound.name`](/docs/http/traffic-policy/index.mdx)                                    | String                                                                       | `policy` has been renamed to `traffic_policy` in agent version 3.14.0.                                                                                                                                                                                                                            |
+| [`policy.outbound.expressions`](/docs/http/traffic-policy/expressions/index.mdx)                 | Array of Strings                                                             | `policy` has been renamed to `traffic_policy` in agent version 3.14.0.                                                                                                                                                                                                                            |
+| [`policy.outbound.actions.type`](/docs/http/traffic-policy/actions/index.mdx)                    | String                                                                       | `policy` has been renamed to `traffic_policy` in agent version 3.14.0.                                                                                                                                                                                                                            |
+| [`policy.outbound.actions.config`](/docs/http/traffic-policy/actions/index.mdx)                  | custom                                                                       | `policy` has been renamed to `traffic_policy` in agent version 3.14.0.                                                                                                                                                                                                                            |
```

Comment:
the original thought was that we have the old policy stuff as well so that people using older agents can still configure policy. However, maybe it makes sense to delete it now that we are doing the full changeover. 

My only concern would be if people with old agents come to this page, they'd try using `traffic_policy` and not understand why it isn't working...maybe I stick a warning at the top of the table?

Reply:
at some point, it would be nice if our docs were versioned with the agent version. but that's a huge undertaking :sigh: